azure/iac/blobservices.rego

```diff
@@ -66,13 +76,23 @@ azure_issue["storage_blob_container_soft_delete"] {
 
 storage_blob_container_soft_delete {
     lower(input.resources[_].type) == "microsoft.storage/storageaccounts/blobservices"
+    not azure_attribute_absence["storage_blob_container_soft_delete"]
     not azure_issue["storage_blob_container_soft_delete"]
 }
 
+storage_blob_container_soft_delete = false {
+    azure_attribute_absence["storage_blob_container_soft_delete"]
+}
+
+
 storage_blob_container_soft_delete = false {
     azure_issue["storage_blob_container_soft_delete"]
 }
 
+storage_blob_container_soft_delete_miss_err = "Soft delete on blob service container is not exist" {
```

Comment:
@shahinshirvani `Soft delete on blob service container should be enabled`

Reply:
@shahinshirvani add this message in the else block of `storage_blob_container_soft_delete_err`  for `azure_attribute_absence["storage_blob_container_soft_delete"]`

**Message can be:** `microsoft.storage/storageaccounts/blobservices` resource property `containerDeleteRetentionPolicy.enabled` is missing.

**See disk.rego for example** 
```
disk_encrypt_err = "microsoft.compute/disks resoruce property encryptionSettingsCollection.enabled is missing" {
    azure_attribute_absence["disk_encrypt"]
} else = "Azure disk currently does not have Azure Disk Encryption (ADE) enabled" {
    azure_issue["disk_encrypt"]
}
```

CC: @farchide

Reply:
fixed